src/app/components/AlertBox/index.tsx

```diff
@@ -61,11 +62,18 @@ export function AlertBox(props: Props) {
       background={{ color: mapStatus[props.status].background }}
       pad={{ horizontal: 'small' }}
     >
-      <Box direction="row" gap="small" align="center" justify={props.center ? 'center' : 'start'}>
-        {props.icon && <Text color={color}>{props.icon}</Text>}
-        <Text weight="bold" size="12px" style={{ marginTop: 10, marginBottom: 10 }}>
-          {props.children}
-        </Text>
+      <Box
+        direction="row"
+        justify={props.justify ?? 'start'}
+        pad={{ horizontal: 'none', vertical: 'xsmall' }}
+      >
+        <Box direction="row" gap="small" align="center">
+          {props.icon && <Text color={color}>{props.icon}</Text>}
+          <Text size="xsmall" weight="bold" style={{ verticalAlign: 'middle' }}>
```

Comment:
This is not needed as parent flexbox container takes care of alignment, right?

Reply:
Since I replaced `size="12px"` with `size="xsmall"`, the later adds `line-height` props. Thats why I wanted to make it centered - otherwise looks weird with different vertical paddings inside alert. It is not ideal, as you would have to do the same on the `endSlot`, but I just disliked that `size` was hardcoded.